shed/event_streams.py

```diff
@@ -250,7 +250,9 @@ def curate_streams(self, nds):
         else:
             names, docs = nds
             name = names
-            docs = (docs,)
+            # make sure it's a dict not tuple
+            if not isinstance(nds[1], tuple):
```

Comment:
@CJ-Wright in particular, i think this should allow nested calls of `curate_streams`